src/main/res/drawable/standard_button.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android" android:shape="rectangle">
+  <solid android:color="#D6D7D7" />
```

Comment:
Is this related to this feature? 
Also, the button ends up looking like this on Android 6: 
![image](https://user-images.githubusercontent.com/35681649/154920136-1e53aba8-9e96-4efb-8dc8-7ebf66babe97.png)

White text on light grey is hard to see.

Reply:
I'll fix this button. Is the screen background dark gray because of the phone theme? Was it like that before? I wonder if that's a regression as well.

Reply:
The background was always like this, as far as I remember.

Reply:
I've fixed the button